src/modules/rbd/rbd-list/rbd-list.js

```diff
@@ -90,8 +90,11 @@
                     rbd.clusterName = clusterObj.cluster_name;
                 }
                 if (typeof list[i].used !== "undefined" && typeof list[i].provisioned !== "undefined") {
+                    rbd.utilization = {};
                     utilization_percent = ((parseInt(list[i].used) * 100) / parseInt(list[i].provisioned));
-                    rbd.utilization = { "percent_used": utilization_percent };
+                    rbd.utilization["used"] = parseInt(list[i].used);
```

Comment:
@cloudbehl Remove parseInt function. Donut component is parsing the used and total properties.